libs/wire-api/src/Wire/API/Call/TURN.hs

```diff
@@ -93,36 +100,71 @@ import Wire.API.Arbitrary (Arbitrary (arbitrary), GenericUniform (..))
 -- | A configuration object resembling \"RTCConfiguration\"
 --
 -- The \"ttl\" field is a proprietary extension
+-- The \"sft_servers\" field is a proprietary extension
 --
 -- cf. https://developer.mozilla.org/en-US/docs/Web/API/RTCPeerConnection/RTCPeerConnection#RTCConfiguration_dictionary
 data RTCConfiguration = RTCConfiguration
-  { _rtcConfIceServers :: List1 RTCIceServer,
+  { _rtcConfIceServers :: NonEmpty RTCIceServer,
+    _rtcConfSftServers :: Maybe (NonEmpty SFTServer),
```

Comment:
we just talked about how this should never go from `Just` to `Nothing`, but now it occurs to me that: why?  what's the semantics of that case, and what do we report to the client?  could use a type wtih more expressive constructors (as you suggested) plus haddocks on what it means.

Reply:
The semantics are we don't want to erase the stale data on a failed lookup. This is why I want to create another type which can make these semantics more obvious.

Reply:
There is new type now. Let me know if it makes sense